app/login.js

```diff
@@ -0,0 +1,23 @@
+function checkUser () {
+
+  var email = document.getElementById( 'email' ).value;
+  var password = document.getElementById( 'password' ).value;
+
+  if ( accountService.checkLogin (email , password) == "1" ) {
```

Comment:
pls put the result of accountService.checkLogin into a single var at the begining of the function. Please use error code constants here instead of strings